README.md

```diff
@@ -37,117 +37,168 @@ To make sure this is fast, please use a work email and have someone from your co
 
 #### Requirements
 
-- The latest [Xcode](https://developer.apple.com/xcode/).
-- PSPDFKit 10.2.1 for iOS or later
-- react-native >= 0.63.4
-- CocoaPods >= 1.10.1
+- A [development environment](https://reactnative.dev/docs/environment-setup) for running React Native projects using the React Native CLI (not the Expo CLI)
+- The [latest stable version of Xcode](https://developer.apple.com/xcode/)
+- The [latest version of PSPDFKit for iOS](https://pspdfkit.com/changelog/ios/)
+- The [latest stable version of CocoaPods](https://github.com/CocoaPods/CocoaPods/releases). If you don’t already have CocoaPods installed, follow the [CocoaPods installation guide](https://guides.cocoapods.org/using/getting-started.html#installation) to install CocoaPods on your Mac.
 
 #### Getting Started
 
 Let's create a simple app that integrates PSPDFKit and uses the `react-native-pspdfkit` module.
 
-1. Make sure `react-native-cli` is installed: `yarn global add react-native-cli`
-2. Create the app with `react-native init YourApp`
-3. Step into your newly created app folder: `cd YourApp`
-4. Install `react-native-pspdfkit` from GitHub: `yarn add github:PSPDFKit/react-native`
-5. Install all the dependencies for the project: `yarn install`. (Because of a [bug](https://github.com/yarnpkg/yarn/issues/2165) you may need to clean `yarn`'s cache with `yarn cache clean` before.)
-6. Open `ios/Podile` in a text editor: `open ios/Podfile`, update the platform to iOS 12, and add the PSPDFKit podspecs:
+1. In the terminal app, change the current working directory to the location you wish to save your project. In this example, we’ll use the `~/Documents/` directory:
+
+```bash
+cd ~/Documents
+```
+
+2. Create the React Native project by running the following command:
+
+```bash
+react-native init PSPDFKitDemo
+```
+
+3. In the terminal app, change the location of the current working directory inside the newly created project:
+
+```bash
+cd PSPDFKitDemo
+```
+
+4. Add the PSPDFKit plugin: 
+
+```bash
+yarn add github:PSPDFKit/react-native
+```
+
+5. Install all the dependencies for the project: 
+
+```bash
+yarn install
+```
+
+6. Open your project’s Podfile in a text editor to update the platform to iOS 12, and add the PSPDFKit Podspec: 
+
+```bash
+open ios/Podfile
+```
+
+Your Podfile should look like this:
 
 ```diff
 require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/react-native/scripts/react_native_pods'
```

Comment:
This is listed twice.
I guess by accident?

```suggestion
```

Reply:
Yes. It was an accident 🙈